sklearn/datasets/tests/test_covtype.py

```diff
@@ -6,9 +6,9 @@
 from sklearn.datasets.tests.test_common import check_return_X_y
 
 
-def test_fetch(fetch_covtype_fxt):
-    data1 = fetch_covtype_fxt(shuffle=True, random_state=42)
-    data2 = fetch_covtype_fxt(shuffle=True, random_state=37)
+def test_fetch(fetch_covtype_fxt, global_random_seed):
+    data1 = fetch_covtype_fxt(shuffle=True, random_state=global_random_seed)
+    data2 = fetch_covtype_fxt(shuffle=True, random_state=global_random_seed)
```

Comment:
you may use "global_random_seed+1" for example as a different seed

Reply:
No, we don't want to do that. `global_random_seed` is a `pytest` fixture and our CI will try different random seeds on some specific run. Also, we always try the first 100 random seeds before merging thanks to this fixture. @Veghit please refer to the original issue.

Reply:
Looking over the original test, I think it assumes that `data1` and `data2` comes from two different seeds. Thus, `global_random_seed + 1` makes sense in this context.

Reply:
Yes, let's go with ``global_random_seed + 1``